test/core/origin_core/get_accumulate_gas_target.js

```diff
@@ -0,0 +1,86 @@
+// Copyright 2018 OpenST Ltd.
```

Comment:
Currently, if I see it correctly, the naming convention of the test file is to use the function name (`getAccumulateGasTarget`).